src/Microsoft.DotNet.Interactive.HttpRequestParser/HttpRequestNode.cs

```diff
@@ -70,10 +74,9 @@ internal class HttpRequestNode : HttpSyntaxNode
     public HttpBindingResult<HttpRequestMessage> TryGetHttpRequestMessage(HttpBindingDelegate bind)
     {
         var request = new HttpRequestMessage();
-        var diagnostics = new List<Diagnostic>();
-        var success = true;
+        var diagnostics = new List<Diagnostic>(base.GetDiagnostics());
```

Comment:
Should we early out and avoid trying to bind here if `diagnostics` contains (syntax) errors?

Reply:
My thinking was that binding can provide additional and potentially orthogonal errors. It might be reasonable to see errors for both, e.g. a misspelled HTTP verb **_and_** an undeclared variable.

Reply:
I see - that sounds reasonable.